sdk/nodejs/tests/runtime/testdata/closure-tests/test.ts

```diff
@@ -57,8 +57,10 @@ it(`resolve to the correct typescript version within the pulumi package`,
 describe(`closure tests (TypeScript ${typescript.version})`, function () {
     const cases = readdirSync("cases"); // describe does not support async functions
     for (const testCase of cases) {
-        const { func, isFactoryFunction, error: expectedError, description, allowSecrets, after } = require(`./cases/${testCase}`);
-
+        const filesInCase = readdirSync(`./cases/${testCase}`);
+        const testCaseFile = filesInCase.find((file: string) => /^index.(cj|j)s$/.test(file));
+        if (!testCaseFile) throw new Error(`No test file specified for case ${testCase}`)
```

Comment:
```suggestion
        if (!testCaseFile) {
            throw new Error(`No test file specified for case ${testCase}`);        
        }
```
This is a nice quality-of-life fix, thanks! That said, in general it'd be good to do these kinds of things in separate PRs to avoid muddying the waters. Also, do we need to think about `.ts` files here? (Potentially one reason to break this and discuss out if so)

Reply:
Ideally yes - but unfortunately this is a necessary part of (or at least precursor) to the rest of this, as in order to properly test minified JS serialisation I couldn't use .ts files for the test cases, and had to update the test runner accordingly. Hopefully it's a small enough change to the runner that we're comfortable with it, but agree in an ideal world that they would be separate, and i've tried to keep the changes to an absolute minimum.